include/caffe/util/math_functions.hpp

```diff
@@ -33,6 +33,16 @@ template <typename Dtype>
 void caffe_cpu_axpby(const int N, const Dtype alpha, const Dtype* X,
     const Dtype beta, Dtype* Y);
 
+// y[i]= max(a*x[i], b*y[i])
```

Comment:
Please space-pad operators, even in comments (`y[i] = max(a * x[i], b * y[i])`).